tests/testthat/test-time_bins.R

```diff
@@ -1,5 +1,6 @@
 test_that("time_bins() works", {
-
+  # Skip if offline
+  skip_if_offline(host = "macrostrat.org")
```

Comment:
We should either put the macrostrat related tests at the very end and only skip those or put them all within a separate `test_that()`.